docs/manuals/source/TasksAndConcepts/Plugins.rst

```diff
@@ -591,6 +591,391 @@ After the restore process has finished, the restored VMDK files can be found und
+
+To verify SSL certificates, the plugin must know the CA certificate of the oVirt enviroment,
+it can be downloaded from the oVirt/RHV engine start page manually, or by using the following
+command:
+
+:command:`curl -k -o /etc/bareos/ovirt-ca.cert https://engine.example.com/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`
+
+For each VM to be backed up, a **job** and a **fileset** must be configured. For
+example to backup the VM **testvm1**, configure the fileset as follows:
+
+.. code-block:: bareosconfig
+   :caption: /etc/bareos/bareos-dir.d/fileset/testvm1_fileset.conf
+
+   FileSet {
+      Name = "testvm1_fileset"
+
+      Include {
+         Options {
+            signature = MD5
+            Compression = LZ4
+         }
+         Plugin = "python:module_path=/usr/lib64/bareos/plugins:module_name=bareos-fd-ovirt:ca=/etc/bareos/ovirt-ca.cert:server=engine.example.com:username=admin@internal:password=secret:vm_name=testvm1"
+      }
+   }
+
+.. note::
+
+   The Plugin options string can currently not be split over multiple lines in the configuration file.
+
+And the job as follows:
+
+.. code-block:: bareosconfig
+   :caption: /etc/bareos/bareos-dir.d/job/testvm1_job.conf
+
+   Job {
+      Name = "testvm1_job"
+      JobDefs = "DefaultJob"
+      FileSet = "testvm1_fileset"
+   }
+
+Mandatory Plugin Options:
+
+module_path
+   Path to the plugin, when installed from Bareos packages, this is always
+   :file:`/usr/lib64/bareos/plugins`
+
+module_name
+   Always :file:`bareos-fd-ovirt`
+
+ca
+   Path to the oVirt/RHV SSL CA File, the CA File must be downloaded as described above
+
+server
+   The FQDN of the oVirt/RHV engine server
+
+username
+   The username of an account which has appropriate privileges
+
+password
+   The password for the user that is configured with **username**
+
+vm_name
+   The name of the VM to be backed up
+
+storage_domain
+   The target storage domain name (only for restore)
+
+Optional Plugin Options:
+
+uuid
+   Instead of specifying the VM to be backed up by name (using option **vm_name**), the VM
+   can be specified by its uuid
+
+include_disk_aliases
+   Comma separated list of disk alias names to be included only, if not specified, all disks
+   that are attached to the VM are included. Currently only used on backup.
+
+exclude_disk_aliases
+   Comma separated list of disk alias names to be excluded, if not specified, no disk will
+   be excluded. Currently only used on backup. Note that the **include_disk_aliases** options
+   is applied first, then **exclude_disk_aliases**, so using both usually makes no sense.
+   Also note that disk alias names are not unique, so if two disks of a VM have the same
+   alias name, they will be excluded both.
+
+overwrite
+   When restoring disks of an existing VM, the option **overwrite=yes** must be explictly
+   passed to prevent from accidentally overwriting an existing VM.
```

Comment:
as mentioned, revise for clearness: if overwrite=off it will prevent, overwrite=on will force to overwrite

Reply:
Changed the text to clarify this.